tezos_interop/tezos_interop.re

```diff
@@ -555,29 +555,26 @@ module Fetch_storage: {
       )
     };
   };
-
-  let _run = (~rpc_node, ~confirmation, ~contract_address) => {
+  let run = (~rpc_node, ~confirmation, ~contract_address) => {
     let input = {
       rpc_node: Uri.to_string(rpc_node),
       confirmation,
       contract_address: Address.to_string(contract_address),
     };
     // TODO: stop hard coding this
     let command = "node";
-    let.await file =
-      Lwt_io.with_temp_file(~suffix=".js", ((file, oc)) => {
-        let.await () = Lwt_io.write(oc, [%blob "fetch_storage.bundle.js"]);
-        await(file);
-      });
-    let.await output =
-      Lwt_process.pmap(
-        (command, [|command, file|]),
-        Yojson.Safe.to_string(input_to_yojson(input)),
-      );
-    switch (Yojson.Safe.from_string(output) |> output_of_yojson) {
-    | Ok(storage) => await(Ok(storage))
-    | Error(error) => await(Error(error))
-    };
+    Lwt_io.with_temp_file(~suffix=".js", ((file, oc)) => {
+      let.await () = Lwt_io.write(oc, [%blob "fetch_storage.bundle.js"]);
+      let.await output =
+        Lwt_process.pmap(
+          (command, [|command, file|]),
+          Yojson.Safe.to_string(input_to_yojson(input)),
+        );
+      switch (Yojson.Safe.from_string(output) |> output_of_yojson) {
+      | Ok(storage) => await(Ok(storage))
+      | Error(error) => await(Error(error))
+      };
+    });
```

Comment:
I don't think this change should be here, also I noticed a problem that this file is being created on every execution and not in a global level.

Reply:
Addressed by rebasing in the next commit